taxonomy/emsi/client.py

```diff
@@ -138,13 +142,12 @@ def get_skill_details(self, skill_id):
         Returns:
             (dict): A dictionary containing the skill details.
         """
-        path = 'skills/{skill_id}'.format(
-            skill_id=skill_id,
-        )
         try:
-            endpoint = getattr(self.client, path)
-            return endpoint().get()
-        except (SlumberBaseException, ConnectionError, Timeout) as error:
+            api_url = self.get_api_url(f'skills/{skill_id}')
```

Comment:
nit: sending the variable "path" as is would to this method would look a lot more neater.